.repo-metadata.json

```diff
@@ -10,5 +10,6 @@
   "repo": "googleapis/python-cloudbuild",
   "distribution_name": "google-cloud-build",
   "api_id": "cloudbuild.googleapis.com",
+  "codeowner_team": "@googleapis/cicd",
```

Comment:
@Shabirmean The python repos don't currently template the CODEOWNERS file, so you'll also need to add the team to https://github.com/googleapis/python-cloudbuild/blob/master/.github/CODEOWNERS

Reply:
Ah ignore me, I see the team is already in the list. :) You might want to also add `cicd` to `/samples`

Reply:
@busunkim96 - they seem to have been [updated already](https://github.com/googleapis/python-cloudbuild/blob/master/.github/CODEOWNERS#L8); so I left them as is

Reply:
@busunkim96 Done. Thank you for reviewing 🙌🏼